packages/opencensus-web-instrumentation-zone/src/on-page-interaction.ts

```diff
@@ -47,7 +47,7 @@ export class OnPageInteractionStopwatch {
     const rootSpan = this.data.rootSpan;
     rootSpan.addAttribute('EventType', this.data.eventType);
     rootSpan.addAttribute('TargetElement', this.data.target.tagName);
-    rootSpan.addAttribute(ATTRIBUTE_HTTP_URL, location.href);
+    rootSpan.addAttribute(ATTRIBUTE_HTTP_URL, this.data.originLocation);
```

Comment:
Smart! Seems you do need to keep capturing `href` in the data for each interaction zone in addition to path and hash I think.